ebpf/state.go

```diff
@@ -308,7 +314,7 @@ func (ns *networkState) cleanupState(now time.Time, clearExpiredStats, flushStat
 		if c.lastFetch.Add(ns.expiry).Before(now) {
```

Comment:
Should `lastFetch` be using the same time source too ?

Reply:
Not it's fine not to, because this is for fetches by another userspace program.